Add Kurdish language option to UI

BudsieBuds opened the pull request:
Makes the Kurdish language selectable in the UI. As requested here: #4209.

@jekkos I do wonder why this language was added as ckb_IQ instead of the ISO 639-1 standard used for all other languages, in this case being `ku`. If you decide to change this for consistency, you should also change the following:

app/Config/App.php
`ckb_IQ',` to `'ku',`

app/Helpers/locale_helper.php
`'ckb_IQ:centralkurdish' => 'Central Kurdish (Iraq)',` to `'ku:kurdish' => 'Kurdish',`

BudsieBuds commented:
As is, this errors when the language is selected. Probably because of the underscore in the language name.

jekkos commented:
Thanks for the PR. I added the language in weblate , so it is probably a setting there. Would need to check if it is able to use what you suggest here instead

BudsieBuds commented:
Like this it works and follows the ISO naming convention. Changed from `ckb_IQ` to `ku` fixed the error. Had to add the empty keys to solve new errors. Also added some missing language files.

Do you have to change something in Weblate for this to show up? Or does it auto update if merged?

jekkos commented:
Thanks for the fixes. Now I can pull the code back into weblate. Then I'll need to check if the files end up in the correct place.

BudsieBuds commented:
@jekkos I overlooked this and assumed it was Kurdish. After some research, I found that adding the country code (ISO 3166) isn’t necessary. Since Central Kurdish lacks a two-letter code under ISO 639-1, I've switched to using the three-letter code from ISO 639-3 instead. The country code may be added in the future if there are more variants. Updated for merging here: #4211. 👍

OmerQadirFM commented:
Kurds primarily speak three different dialects:  
kmr – Northern Kurdish (it might be worth considering including this if someone requests it),  
ckb – Central Kurdish (this is the dialect I speak),  
sdh – Southern Kurdish. 

As for country codes, they are not necessary since the same dialects are spoken across different countries.

# Conclusion
kmr Kurdish (Northern)
ckb - Kurdish (Central)

OmerQadirFM commented:
To ensure easier finding, change it from Central Kurdish to Kurdish (Central).

![image](https://github.com/user-attachments/assets/b9223de5-b9c4-4d96-90ec-6e3564fb1f6b)

BudsieBuds commented:
I chose the term Central Kurdish because it is used by the Language Coding Agency for ISO 639-3. See here: https://iso639-3.sil.org/code/ckb

OmerQadirFM commented:
Initially, when switching to Kurdish, users often scroll down to the letter K to find it. It would be much easier to rename it as Kurdish (Central).

BudsieBuds commented:
I reexamined the naming conventions and have updated several language names to enhance discoverability. Changes will be live if #4218 is merged.
